docs/docs/tutorial.md

```diff
@@ -677,6 +677,7 @@ var CommentBox = React.createClass({
   },
   handleCommentSubmit: function(comment) {
     var comments = this.state.data;
+    comment.id = Date.now();
```

Comment:
Can you add a comment saying that this `id` is temporary and will be replaced by the server as soon as the server returns a response?